azure-pipelines.yml

```diff
@@ -29,80 +30,97 @@ pr:
+                     /p:ManifestBuildBranch=$(Build.SourceBranchName)
+                     /p:ManifestBuildNumber=$(Build.BuildNumber)
+
+        steps:
+        - checkout: self
+          clean: true
+        - task: NuGetCommand@2
+          displayName: 'Clear NuGet caches'
+          condition: succeeded()
+          inputs:
+            command: custom
+            arguments: 'locals all -clear'
+        - script: eng\common\cibuild.cmd -configuration $(_BuildConfig) -prepareMachine $(_BuildArgs)
+          displayName: Build and Publish
+        - task: PublishBuildArtifacts@1
+          displayName: Upload TestResults
+          condition: always()
+          continueOnError: true
+          inputs:
+            pathtoPublish: artifacts/TestResults/$(_BuildConfig)/
+            artifactName: $(Agent.Os)_$(Agent.JobName) TestResults
+            artifactType: Container
+            parallel: true
+        - task: PublishBuildArtifacts@1
+          displayName: Upload package artifacts
+          condition: and(succeeded(), eq(variables['system.pullrequest.isfork'], false), eq(variables['_BuildConfig'], 'Release'))
+          inputs:
+            pathtoPublish: artifacts/packages/
+            artifactName: packages
+            artifactType: Container
+            parallel: true
 
-      steps:
-      - checkout: self
-        clean: true
-      - task: NuGetCommand@2
-        displayName: 'Clear NuGet caches'
-        condition: succeeded()
-        inputs:
-          command: custom
-          arguments: 'locals all -clear'
-      - script: eng\common\cibuild.cmd -configuration $(_BuildConfig) -prepareMachine $(_BuildArgs)
-        displayName: Build and Publish
-      - task: PublishBuildArtifacts@1
-        displayName: Upload TestResults
-        condition: always()
-        continueOnError: true
-        inputs:
-          pathtoPublish: artifacts/TestResults/$(_BuildConfig)/
-          artifactName: $(Agent.Os)_$(Agent.JobName) TestResults
-          artifactType: Container
-          parallel: true
-      - task: PublishBuildArtifacts@1
-        displayName: Upload package artifacts
-        condition: succeeded()
-        inputs:
-          pathtoPublish: artifacts/packages/
-          artifactName: packages
-          artifactType: Container
-          parallel: true
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - template: eng\common\templates\post-build\post-build.yml
+    parameters:
+      # Disabled because this repo is still running into issues with signing validation, and the exclusions file doesn't work.
+      enableSigningValidation: false
+      # Shipped package contains minimal managed code and lots of repackaged external and native assemblies. We
+      # don't ship relevant PDBs except (for some assemblies) later in aspnet/AspNetcore.
+      enableSourceLinkValidation: false

```

Comment:
@dougbu I'm only disabling Symbol Validation in my PRs, should I be disabling SourceLink/Signing validation as well?

Reply:
If those checks fail for good reasons, yes. Otherwise, no.